### PR TITLE
Event RESTs and error handling

### PR DESCRIPTION
**Pull request by karomnii**

### CODE REVIEW

File: backend/src/main/java/backend/lezaczek/Services/EventService.java
Comment (nbalicka35) [inline backend/src/main/java/backend/lezaczek/Services/EventService.java:1]: there's no data validation inside eventService.updateEvent(), values are straigh away assigned

File: backend/src/main/java/backend/lezaczek/Helpers/JwtTokenHelper.java
Comment (nbalicka35) [inline backend/src/main/java/backend/lezaczek/Helpers/JwtTokenHelper.java:1]: "no cookie" is kinda short feedback

**Review by nbalicka35 — COMMENTED** [review]
left a single comments

File: backend/src/main/java/backend/lezaczek/Services/EventService.java
Comment (karomnii) [reply backend/src/main/java/backend/lezaczek/Services/EventService.java:1]: Added events validation

File: backend/src/main/java/backend/lezaczek/Helpers/JwtTokenHelper.java
Comment (karomnii) [reply backend/src/main/java/backend/lezaczek/Helpers/JwtTokenHelper.java:1]: changed error message

**Review by nbalicka35 — APPROVED** [review]

**Pull request merged**